source/examples/cmn/cmn-cue_layer.txt

```diff
@@ -0,0 +1,50 @@
+<score>
+  <scoreDef>
+    <staffGrp>
+      <staffDef n="1" lines="5">
+        <clef shape="G" line="2" />
+        <keySig sig="5f" />
+        <meterSig count="3" unit="4" />
+      </staffDef>
+    </staffGrp>
+  </scoreDef>
+  <section>
+    <measure metcon="false">
+      <staff n="1">
+        <layer n="1">
+          <rest dur="4" />
+        </layer>
+        <layer n="2" cue="true">
+          <clef shape="F" line="4" />
+          <note dur="4" oct="2" pname="f" />
+        </layer>
+      </staff>
+    </measure>
+    <measure n="1">
+      <staff n="1">
+        <layer n="1">
+          <mRest />
+        </layer>
+        <layer n="2" cue="true">
```

Comment:
I'm wondering if we should recommend that `cue` should also use `sameas` and point to the cued layer, rather than encoding the layer twice. Thoughts?

Reply:
I've run into this recently. I would say it should be optional, but not necessarily recommended. The cue notes / layer is still an entity in itself, even though it has a semantic connection to some other notes / layer and is derived from there. It could have incorrect notes, though. In that case, one may perhaps prefer a `@corresp` over `@sameas`?

Reply:
In the case of a cued layer, I would expect that a full layer would be cued. Otherwise, we would just mark four notes as cues.

Reply:
Yes, we can point to the original layer, if we have a full score encoded. If we have only a single part, where should it point at?

Reply:
Your addition looks great! Thanks.